lib/onelogin/ruby-saml/response.rb

```diff
@@ -727,7 +727,7 @@ def validate_issuer
       # @return [Boolean] True if the SessionNotOnOrAfter of the AuthnStatement is valid, otherwise (when expired) False if soft=True
       # @raise [ValidationError] if soft == false and validation fails
       #
-      def validate_session_expiration(soft = true)
```

Comment:
I believe we can simply remove the param

Reply:
@pitbulk removed!:sparkles: